src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import { User } from '../../types/user';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a key={user.id} className="UserInfo" href={`mailto:${user.email}`}>
```

Comment:
key is redudant here
```suggestion
    <a className="UserInfo" href={`mailto:${user.email}`}>
```